app/scripts/modules/core/src/pipeline/config/stages/deploy/deployExecutionDetails.html

```diff
@@ -20,10 +20,7 @@
           </div>
           <dt>Cluster</dt>
           <dd>{{stage.context | clusterName }}</dd>
-        </dl>
-      </div>
-      <div class="col-md-6">
-        <dl class="dl-narrow dl-horizontal">
+
```

Comment:
i love random whitespace

Reply:
FINE

Reply:
It was there as a reminder that this was once two columns but FINE I'll remove it since nobody cares but me.